packages/migrate/test/index.ts

```diff
@@ -1,8 +1,14 @@
-const Migrate = require("../index");
-const Config = require("@truffle/config");
-const assert = require("assert");
-const sinon = require("sinon");
-let options, migrations;
+import Migrate from "../src";
+import Config from "@truffle/config";
+import assert from "assert";
+import * as sinon from "sinon";
+import { Migration } from "../src/Migration";
+let options: Config, migrations: Migration[], fakeMigration: Migration;
+
+const unstub = (stubbedThing: object, methodName: string) => {
+  // @ts-ignore
+  stubbedThing[methodName].restore();
+};
```

Comment:
I like the idea of unstub to pair up with sinon.stub.  This makes the code easy to follow.

Reply:
For some reason `sinon` stubs don't seem to play well with TypeScript. I actually made this method so I would only have to write `// @ts-ignore` once instead of 20 times lol